src/app.py

```
@@ -0,0 +1,435 @@
+            messagebox.showinfo("Информация", "Файлы загружены! Генерация графиков...")
+            
+            sample_plot_t = self.create_plots(t_data, 't')
+            sample_plot_r = self.create_plots(r_data, 'r')
+            if self.first_input:
+                self.create_plot_areas()
+                self.first_input = False
+
+            self.display_plots(sample_plot_t, "translation")
+            self.display_plots(sample_plot_r, "rotation")
+            
+        except Exception as e:
```

Ruff (0.14.8)

Do not catch blind exception: `Exception`

(BLE001)